torch/_refs/__init__.py

```diff
@@ -3630,7 +3628,8 @@ def arange(
             # pin_memory=pin_memory,
             requires_grad=requires_grad,
         )
-    elif a is not None and b is None:
+    else:
+        assert a is not None and b is None
```

Comment:
Update this to use `utils.check()`

Reply:
(Also this should add a comment as to what this clause is for)

Reply:
The if/else pattern here seems a little awkward to me vs. just populating the `start` value if not provided and then going through one code path

Reply:
Wait... does this implement the logic correctly? 

```
torch.arange(end=5)
```

Is valid -- how would that work here?

Reply:
Yeah, this logic felt weird, and it is indeed incorrect. I'll fix it on Monday.